source/development/packaging/generate-dashboard-package.rst

```diff
@@ -0,0 +1,186 @@
+portability. The building process is self-contained in the application
+code.
+
+Build manually
+^^^^^^^^^^^^^^
+
+1. To use the scrip you first need to generate the packages from the repositories:
+
+- ``wazuh-dashboard``
+- ``wazuh-security-dashboards-plugin`` 
+- ``wazuh-dashboard-plugins``
+
+To do so, follow these steps:
+
+   1.1. Clone the Wazuh Dashboard repository and build the application.
+
+   .. code:: console
+
+      # git clone -b <branch> https://github.com/wazuh/wazuh-dashboard.git
+      # cd wazuh-dashboard/
+      # yarn osd bootstrap
+      # yarn build --linux --skip-os-packages --release
+
+   1.2. Clone the Wazuh Security Dashboards Plugin and build the plugin.
+
+   .. code:: console
+
+      # cd plugins/
+      # git clone -b <branch> https://github.com/wazuh/wazuh-security-dashboards-plugin.git
+      # cd wazuh-security-dashboards-plugin/
+      # yarn
+      # yarn build
+
+   1.3. Clone the Wazuh Dashboard Plugins repository and build the plugins.
+
+   .. code:: console
+
+      # cd ../
+      # git clone -b <branch> https://github.com/wazuh/wazuh-dashboard-plugins.git
+      # cd wazuh-dashboard-plugins/
+      # cp -r plugins/* ../
+      # cd ../main
+      # yarn
+      # yarn build
+      # cd ../wazuh-core/
+      # yarn
+      # yarn build
+      # cd ../wazuh-check-updates/
+      # yarn
+      # yarn build
+
+   1.4. Zip the packages and move them to the packages folder
+
+   .. code:: console
+
+      # cd ../../../
+      # mkdir packages
+      # cd packages
+      # zip -r -j ./dashboard-package.zip ../wazuh-dashboard/target/opensearch-dashboards-<opensearch version>-linux-x64.tar.gz
+      # zip -r -j ./security-package.zip ../wazuh-dashboard/plugins/wazuh-security-dashboards-plugin/build/security-dashboards-<opensearch version>.0.zip
+      # zip -r -j ./wazuh-package.zip ../wazuh-dashboard/plugins/wazuh-check-updates/build/wazuhCheckUpdates-<opensearch version>.zip ../wazuh-dashboard/plugins/main/build/wazuh-<opensearch version>.zip ../wazuh-dashboard/plugins/wazuh-core/build/wazuhCore-<opensearch version>.zip
+
+At this point you must have 3 packages in the packages folder:
+
+-  ``dashboard-package.zip``
+-  ``security-package.zip``
+-  ``wazuh-package.zip``
+
+2. Generate the final packages, by running the script ``build-packages.sh`` in the ``dev-tools/build-packages/`` folder of the repository. 
+The script requires the following parameters:
+
+- ``-v``: Version of the package.
+- ``-r``: Revision of the package.
+- ``--deb`` or ``--rpm``: Distribution of the package.
+- ``-a``: Path to the ``wazuh-package.zip``.
+- ``-s``: Path to the ``security-package.zip``.
+- ``-b``: Path to the ``dashboard-package.zip``.
+
+.. code:: console
+
+   # cd ../wazuh-dashboard/dev-tools/build-packages/
+   # ./build-packages.sh -v <version> -r <revision> <distribution(--deb || --rpm)> -a <path to wazuh-package.zip> -s <path to security-package.zip> -b <path to dashboard-package.zip>
```

Comment:
Does these paths require `file:///` at the beginning? If so, we should specify that